be/src/runtime/vectorized_row_batch.h

```diff
@@ -139,7 +139,7 @@ class VectorizedRowBatch {
     void dump_to_row_block(RowBlock* row_block);
 
 private:
-    const TabletSchema* _schema;
+    const std::vector<TabletColumn>* _schema;
```

Comment:
Only in RowCursor, I must to use std::vector<TabletColumn>. Because there exists a situation it only partial TabletColumn. Otherwise, TabletSchema is only parameter can be imported.

Reply:
OK